### PR TITLE
Add CreateERC721Controllable event 

### PR DESCRIPTION
**Pull request by aodhgan**

### CODE REVIEW

File: test/LootBoxController.test.js [hunk 1]
```diff
@@ -56,6 +56,12 @@ describe('LootBoxController', () => {
     let lootBoxControllerResult = await deployments.get('LootBoxController')
```
Comment (asselstine) [inline test/LootBoxController.test.js]: Above this line- let's replace all of the `deployContract` calls with code that pulls in the already deployed ERC721, ERC20 and ERC1155

File: contracts/ERC721ControlledFactory.sol [hunk 2]
```diff
@@ -44,6 +48,7 @@ contract ERC721ControlledFactory {
     ERC721Controlled result = ERC721Controlled(payable(Create2.deploy(0, _salt(msg.sender), erc721ControlledBytecode)));
     nonces[msg.sender] = nonces[msg.sender].add(1);
     result.initialize(name, symbol, baseURI, msg.sender);
+    emit ERC721ControlledCreated(address(result));
```
Comment (asselstine) [inline contracts/ERC721ControlledFactory.sol:51]: Let's make sure to add an expectation for this event in the test
`
await expect(factory.createERC721Controlled(....)).to.emit(factory, 'ERC721ControlledCreated')
`

File: test/LootBoxController.test.js [hunk 3]
```diff
@@ -56,6 +56,12 @@ describe('LootBoxController', () => {
...
     lootBoxController = await buidler.ethers.getContractAt('LootBoxController', lootBoxControllerResult.address, wallet)
 
+
+  // insert equivilent erc20mintable, erc1155 erc721
+  let ERC20Mintable
+  let ERC20MintableResult = await deployments.get('ERC20Mintable')
+  ERC20Mintable = await buidler.ethers.getContractAt('ERC20Mintable', ERC20MintableResult.address, wallet)
```
Comment (asselstine) [inline test/LootBoxController.test.js]: What is the `ERC20Mintable` var used for?  Perhaps you meant to update the `deployContracts` above

**Review by asselstine — CHANGES_REQUESTED** [review]

File: test/LootBoxController.test.js (hunk 3 above)
Comment (aodhgan) [reply test/LootBoxController.test.js]: Yes, this code should have been removed/serves no purpose.

File: test/LootBoxController.test.js [hunk 4]
```diff
@@ -44,11 +44,19 @@ describe('LootBoxController', () => {
...
+    erc721MintableDeploy = await deployments.get('ERC721Mintable')
+    erc721Mintable = await buidler.ethers.getContractAt('ERC721Mintable', erc721MintableDeploy.address, wallet)
+
+    erc777MintableDeploy = await deployments.get('ERC777Mintable')
+    erc777Mintable = await buidler.ethers.getContractAt('ERC777Mintable', erc777MintableDeploy.address, wallet)
+  
+    erc1155MintableDeploy = await deployments.get('ERC1155Mintable')
+    erc1155Mintable = await buidler.ethers.getContractAt('ERC1155Mintable', erc1155MintableDeploy.address, wallet)
+
+    lootBox = await deployContract(wallet, LootBox, []) // do we want to deploy here or in deloy.js
```
Comment (aodhgan) [inline test/LootBoxController.test.js]: @asselstine do we want to deploy a lootbox here or in deploy.js?

Comment (asselstine) [reply test/LootBoxController.test.js]: Let's delete it.  It doesn't seem to be used anywhere.  Cull all dead code! ☠️

File: contracts/ERC721ControlledFactory.sol [hunk 5]
```diff
@@ -13,6 +13,10 @@ import "./ERC721Controlled.sol";
 contract ERC721ControlledFactory {
   using SafeMath for uint256;
 
+
+  /// @notice Emitted when a ERC721Controlled is created
+  event ERC721ControlledCreated(address);
```
Comment (asselstine) [inline contracts/ERC721ControlledFactory.sol]: The event arg needs a name.  I.e.:

```solidity
event ERC721ControlledCreated(address erc721)
```

**Pull request merged**